CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [5.0.0] (IN PROGRESS)
 
 * bump stripes to 8.0.0 for Orchid/2023-R1. Refs UIPFPAT-52.
+* Add a11y tests
```

Comment:
Please add `Refs UIPFPAT-51` to the end.
`Add a11y tests. Refs UIPFPAT-51.`

Reply:
Done